The MIL-Arkansaw Capital Sized Mech Carrier- Made for the Cult of the Machine God

Rubeebeebee opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
Added the MIL-Arkansaw, a capital sized mech carrier. 

"The Arks are a vital addition to the game’s vessel lineup and should be playable exclusively by members of the Cult of the Machine God. Its capabilities directly mirrors the proficiency and the general capabilities of the Cult. These ships are designed as self-sufficient mother ships, capable of sustaining all needs required to fulfill the Cult’s objectives. Whether serving as an independent Sect or operating under a larger Faction, the Ark allows its crew to act without reliance on external support. They are the navigators of the faith, carrying the Cult’s mission wherever it must go. To support full roleplay immersion and narrative depth, Arks should be available across all factions (Civilian, TSF, USSP, and Rogue) allowing Cult members to integrate seamlessly while remaining true to both Faction and Cult goals." 
- @Dreadwald in discord

It's a huge ship with a dedicated mining and science wing, these wings work in tandem in order to supply the mechbay with research and materials to create mechs. 

in order to facilitate the pilots, a dorms + kitchen area is added. 

for energy, the ship is powered by 3 bananium generators strapped together. Extremely radioactive. Radsuits are provided in order to curb radiation sickness when refueling or doing maintenance. 

of course, to facilitate the worship of the machine god, a shrine area is added. 


## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->

the MIL-Arkansaw is equipped with:
5x 90m autocannons, two of those being on either flank, with one in the front of the bridge. 
8x 20mm PD guns, spread on the broadsides of the ship. 
4x Mining Pulsars, on the tail end of the ship, in order to facilitate mining copious amounts of materials to make mechs and artifacts for research. 

For it's armor, it has two walls layers acting as it's protection, with the outer layer being plastitanium, and the inner wall being  regular reinforced walls. 

It is also equipped with a MS-100 shield generator and the lowest FTL drive available. 

These advantages are offset by the ship's slow speed, both in acceleration and turning, and all thrusters are vulnerable sitting outside on the hull, especially the rear, due to the Arkansaw not having any covering that blindspot, opting to use lattice-hull shields to soak up shots directed at the thruster array on the rear. 

The Arkansaw is outclassed by ships of it's class it terms of both arms and armor, but what it lacks it makes up for in mechs, having three 3x3 mech hangars, and two 4x4 mech hangars respectively. 

These would (hopefully) be filled in with combat mechs to assist the Arkansaw in it's defense or attacks. 

## How to test
<!-- Describe the way it can be tested -->
load in the .yml file in IDE with loadgrid should work fine. 

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->

<img width="431" height="889" alt="Arkansaw Live" src="https://github.com/user-attachments/assets/040cc4a1-e25d-4b61-bf3b-1fd7b00c74b2" />
<img width="1183" height="972" alt="arkansaw bridge+shrine" src="https://github.com/user-attachments/assets/be0d1391-2cd2-447b-97d7-e3d7deb5154d" />
<img width="1162" height="574" alt="arkansaw dorms + engine room" src="https://github.com/user-attachments/assets/8717ad06-2378-43c9-aac9-ae2d0cedc192" />
<img width="1209" height="624" alt="arkansaw mech hangars" src="https://github.com/user-attachments/assets/ff9c2e9c-0c72-4a11-b5d6-bc3e0cf6c3a7" />
<img width="1115" height="393" alt="arkansaw mechbay + medbay" src="https://github.com/user-attachments/assets/f9c2f709-edb2-4e78-9dd4-2847ca12ce7d" />
<img width="1273" height="611" alt="arkansaw science + mining" src="https://github.com/user-attachments/assets/93405d1c-f793-4956-ae19-8c4df85770f6" />
<img width="1263" height="427" alt="arkansaw back end" src="https://github.com/user-attachments/assets/95d8d88d-9f59-4b18-bb1b-c151dfd19922" />


## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
- [X] I can confirm this PR contains no AI-generated content, and did not use any AI-generated content.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->
Theres a problem when sometimes, the R&D console just... refuses to connect to any of the lathes, and i dunno why it happens. Sometimes, it fixes itself. 

<img width="626" height="448" alt="R D2" src="https://github.com/user-attachments/assets/71b7cc6d-abc3-49e2-a501-a28b84c49b2b" />
<img width="758" height="639" alt="ITS FISXED" src="https://github.com/user-attachments/assets/48daeaeb-d0d8-46cf-b420-45c823044055" />


**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->

:cl:
- add: the MIL-Arkansaw, a Cult of the Machine God ARK ship, made to be a capital sized mech carrier.

Rubeebeebee committed:
I want to add another gas mining ship- so here is the HES-Phaeron, a tanker ship with a huge tank, but you'll have to carry either one filtered gas, or unfiltered gas.

Rubeebeebee committed:
i added decals to the twilight

Rubeebeebee committed:
adding the MIL-arkansaw

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee committed:
Merge branch 'Monolith-Station:main' into Arkansaw-Branch

Redrover1760 commented on Resources/Prototypes/_Mono/Shipyard/Expedition/arkansaw.yml:
```diff
@@ -0,0 +1,45 @@
+  access: Mercenary
+  shuttlePath: /Maps/_Mono/Shuttles/Expedition/arkansaw.yml
+  guidebookPage: Null
+  class:
+  - Expedition
```
add additional classes of ships this counts as

Rubeebeebee replied:
> add additional classes of ships this counts as

gotcha, ill add 
- Science
- Carrier (?)
- Mining

Rubeebeebee replied:
done!

Rubeebeebee committed:
changed the designation to EXP and removed the merc access restriction in the EXP console.

added more hardsuit lockers + decals

Added an AI core (ghostrole i joinable) with cameras all across the ship.

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee commented:
added an AI core! it should be a ghostrole join only AI core, so yeah. 

it does say (when you enter the ghostrole) it's an ADS so idk how to change that. 
(i picked AI CORE, VESSEL)
<img width="419" height="456" alt="AI core" src="https://github.com/user-attachments/assets/220cc425-17be-4ee3-97c6-40095d81fc1b" />

<img width="687" height="317" alt="bible" src="https://github.com/user-attachments/assets/4dbd27a0-aa03-4358-9613-87e5cc6d86d8" />

<img width="534" height="965" alt="arkansaw AI view" src="https://github.com/user-attachments/assets/ea55db76-58ed-4c2c-8e3a-85e17974946a" />

Rubeebeebee commented:
> added an AI core! it should be a ghostrole join only AI core, so yeah.
> 
> it does say (when you enter the ghostrole) it's an ADS so idk how to change that. (i picked AI CORE, VESSEL) <img alt="AI core" width="419" height="456" src="https://private-user-images.githubusercontent.com/220646896/467866619-220cc425-17be-4ee3-97c6-40095d81fc1b.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTI4MTY1MDksIm5iZiI6MTc1MjgxNjIwOSwicGF0aCI6Ii8yMjA2NDY4OTYvNDY3ODY2NjE5LTIyMGNjNDI1LTE3YmUtNGVlMy05N2M2LTQwMDk1ZDgxZmMxYi5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcxOFQwNTIzMjlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jZjJlYjg1MThhMDNhOWEyM2Y0NmJjNzliZmJlZDdjMDUyYTA3N2NjNGNmZmNiMTYxN2Y2YmE4NTI3ZmMzY2M3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.h4euZsFEocS5kJwcyBPkrG74iVYMBD17gX_N9sIfyAE">
> <img alt="bible" width="687" height="317" src="https://private-user-images.githubusercontent.com/220646896/467866432-4dbd27a0-aa03-4358-9613-87e5cc6d86d8.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTI4MTY1MDksIm5iZiI6MTc1MjgxNjIwOSwicGF0aCI6Ii8yMjA2NDY4OTYvNDY3ODY2NDMyLTRkYmQyN2EwLWFhMDMtNDM1OC05NjEzLTg3ZTVjYzZkODZkOC5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcxOFQwNTIzMjlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wMzhlNjc5MzA4M2JjNTUyZjRhYmY3NzVmOTIxYjViY2YzYmIwMzNkMjdlNjI1YjlhMGU4MjQyNThjZmUwM2UyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.F4MDj_dBiHUHnfgqVyMC_lwpWkJCB-5Y3wrE6VmjADM"> <img alt="arkansaw AI view" width="534" height="965" src="https://private-user-images.githubusercontent.com/220646896/467866501-ea55db76-58ed-4c2c-8e3a-85e17974946a.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTI4MTY1MDksIm5iZiI6MTc1MjgxNjIwOSwicGF0aCI6Ii8yMjA2NDY4OTYvNDY3ODY2NTAxLWVhNTVkYjc2LTU4ZWQtNGMyYy04ZTNhLTg1ZTE3OTc0OTQ2YS5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcxOFQwNTIzMjlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hZThiNzRlYjdiZDc3NjQ0NWYwZWYzOTM0ZDJmYzBmYWNiNjI3YmRmNjcyOTZhNjJhYzdmNjliNWExYjY5MTQ2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.0cqq63AeTTJ8OHVm6zfxeq485tfyXoqOlJp8ly9WLug">

ive also made it so that the ship is now an EXP designation and removed the merc only access

Rubeebeebee committed:
added additional classes for the arkansaw
- Science
- Mining
- Carrier

Rubeebeebee committed:
Merge branch 'Arkansaw-Branch' of https://github.com/Rubeebeebee/Monolith into Arkansaw-Branch

Redrover1760 committed:
not a ship carrier

monolith8319 committed:
chore: automatically update REUSE headers

Redrover1760 committed:
salvage

Redrover1760 committed:
Merge branch 'Arkansaw-Branch' of https://github.com/rubeebeebee/monolith into pr/Rubeebeebee/1457

Redrover1760 committed:
I hate this thing

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee committed:
added an expedition computer on the salvage bay

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee commented:
added a exped console, yay
<img width="611" height="811" alt="exped bay" src="https://github.com/user-attachments/assets/debab464-55f9-4362-a30d-2f028664d9b9" />

Update: exped works!
<img width="1037" height="695" alt="exped moment" src="https://github.com/user-attachments/assets/2b880947-8022-467d-b466-aeabbb39a275" />
<img width="910" height="778" alt="comically LARGE" src="https://github.com/user-attachments/assets/9eb2f3e4-57a0-425e-bc54-721a6e50db9d" />

Rubeebeebee committed:
fixed the phantom grilles at the AI core shrine

Rubeebeebee committed:
Merge branch 'Arkansaw-Branch' of https://github.com/Rubeebeebee/Monolith into Arkansaw-Branch
